src/main/java/com/faforever/client/rankedmatch/Ladder1v1Controller.java

```diff
@@ -155,6 +167,39 @@ public void initialize() {
 
     JavaFxUtil.addListener(playerService.currentPlayerProperty(), (observable, oldValue, newValue) -> Platform.runLater(() -> setCurrentPlayer(newValue)));
     playerService.getCurrentPlayer().ifPresent(this::setCurrentPlayer);
+    
+    gameService.addOnRankedMatchNotificationListener(message -> {
+      if (message.getQueues() == null) {
+        return;
+      }
+      for (MatchmakerMessage.MatchmakerQueue matchmakerQueue : message.getQueues()) {
+        if (!Objects.equals("ladder1v1", matchmakerQueue.getQueueName())) {
+          continue;
+        }
+        String nextPopTime = matchmakerQueue.getQueuePopTime();
+        if (nextPopTime != null) {
+          nextQueuePopTime = OffsetDateTime.parse(nextPopTime).toInstant();
+        }
+      }
+    });
+    
+    Timeline queuePopTimeUpdater = new Timeline(new KeyFrame(javafx.util.Duration.seconds(1), (ActionEvent event) -> {
+      if (nextQueuePopTime != null) {
+        Instant now = Instant.now();
+        Duration timeUntilPopQueue = Duration.between(now, nextQueuePopTime);
+        if (!timeUntilPopQueue.isNegative()) {
+          String formatted = String.format("%2d:%02d", 
```

Comment:
This format string needs to go into message.properties and it needs to use different format(Chinese time is for example not displayed with : )

Reply:
Ok. Java does not provide locale based duration formatting, thats why this is a string formatter and not a time formatter, and why i assumed that its useless to try to format this. So i guess for chinese durations only the numbers change, but not the minutes/seconds structure of  duration?

Reply:
And if we are dealing with i18n anyway, i may add a "next round: " label as well. Btw. How would a translator know that `%2d:%02d` is supposed to be `minutes : seconds`, whould I comment this in the properties file?

Reply:
No u can do proper formating in the message.properities file  as well. 
![image](https://user-images.githubusercontent.com/19996713/62143554-7e28a980-b2f0-11e9-8975-4dc2e4bb26e1.png) Do something like that

Reply:
then do `i18n.get("timeFormat", minutes, seconds);`

Reply:
Where can i read about this syntax?